.github/workflows/deploy-infra.yaml

```diff
@@ -43,12 +43,13 @@ jobs:
           if [ ! -z "$ERROR" ]
           then
             echo
-            echo Make sure you launch this job using the biomage devops tools, and not from GitHub.
-            echo Launching from GitHub directly is not supported to ensure cluster credentials are not
-            echo kept in the repository.
+            echo This workflow requires some secrets to run. The secrets can be set/rotated
+            echo by running `rotate-ci` from `biomage-utils`. The command has to be run by a
+            echo user with admin privileges for the repository.
```

Comment:
* are you sure the user needs admin privileges? you can't run it?

Reply:
Yup I'm pretty sure. Marcell told me about this, and I checked in `rotate-ci`. The API request to set the secrets require an Auth token. Even without the script, I can't access the admin panel and change the secret. I think the API should have the same auth checking.